fineract-provider/src/main/java/org/apache/fineract/portfolio/account/service/AccountTransfersWritePlatformServiceImpl.java

```diff
@@ -208,6 +211,37 @@ public CommandProcessingResult create(final JsonCommand command) {
         return builder.build();
     }
 
+    @Transactional
+    @Override
+    public CommandProcessingResult adjust(JsonCommand command) {
```

Comment:
Not related change to Savings periodic accrual handling. Please move it into a separate PR!